common.sh

```diff
@@ -61,6 +60,13 @@ build_libfuzzer() {
   mv libFuzzer.a $LIB_FUZZING_ENGINE
 }
 
+# Uses the capability for "fsanitize=fuzzer" in the current clang
+build_fsanitize_fuzzer() {
+  CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address,fuzzer"
```

Comment:
It could only be defined after `build_fuzzer`. Really, all of these vars could just be defined afterward, though. It probably makes the most sense to append here